src/freenet/support/SortedIntSet.java

```diff
@@ -221,12 +191,11 @@ public synchronized void clear() {
 	 * @return sorted array of all items
 	 */
 	public synchronized int[] toArrayRaw() {
```

Comment:
Note that this entire class is deprecated as of this pull request, no need to deprecate every single method as well ☺